libminifi/include/core/ProcessSession.h

```diff
@@ -19,7 +19,6 @@
 #define __PROCESS_SESSION_H__
 
 #include <uuid/uuid.h>
-#include <boost/filesystem.hpp>
```

Comment:
@dtrodrigues thanks for this. I think I let this slip through in a PR despite trying to eliminate boost for those that don't want it. Thanks for catching this and the contribution. I'll get this merged.